data/processed/hesa/gbp_research_income.yaml

```diff
@@ -0,0 +1,22 @@
+api: fetch
+api_documentation: https://www.hesa.ac.uk/data-and-analysis/estates/table-1
+description: Research income received by universities in the NUTS2 region
+endpoint_static: https://www.hesa.ac.uk/data-and-analysis/estates/data.csv
+framework_group: public_rnd_capability
+is_experimental: False
+nuts_level: 2
```

Comment:
We need to add the line:
```
order: [year, nuts_id, nuts_year_spec, value.id]
```
see #131 and #130.

Reply:
Done